lib/stencil-start.js

```diff
@@ -55,7 +55,14 @@ class StencilStart {
         if (cliOptions.variation) {
             await this._themeConfigManager.setVariationByName(cliOptions.variation);
         }
+
         const initialStencilConfig = await this._stencilConfigManager.read();
+
+        // Override Port Number if Cli option Exsist
+        if (cliOptions.port) {
+            initialStencilConfig.port = cliOptions.port;
+        }
+
         // Use initial (before updates) port for BrowserSync
         const browserSyncPort = initialStencilConfig.port;
```

Comment:
I think you can do instead
```suggestion
        const browserSyncPort = cliOptions.port || initialStencilConfig.port;
```

Reply:
then we need do change 

```js
const updatedStencilConfig = this.updateStencilConfig(
            { initialStencilConfig, port: browserSyncPort},
            storeInfoFromAPI,
);